docs/README.md

```diff
@@ -30,10 +30,7 @@
 
 ### Troubleshooting
 
-- [Troubleshooting Detox Installation](Troubleshooting.Installation.md)
-- [Troubleshooting Failing Tests](Troubleshooting.RunningTests.md)
-- [Troubleshooting Synchronization](Troubleshooting.Synchronization.md)
-- [Dealing With Flakiness in Tests](Troubleshooting.Flakiness.md)
+- [Troubleshooting ToC](Troubleshooting.md)
```

Comment:
I don't like this idea. Could you just combine both, e.g.?

### Troubleshooting

- [Table of contents](Troubleshooting.md)
  - [Troubleshooting Detox Installation](Troubleshooting.Installation.md)
  - [Troubleshooting Failing Tests](Troubleshooting.RunningTests.md)
  - [Troubleshooting Synchronization](Troubleshooting.Synchronization.md)
  - [Dealing With Flakiness in Tests](Troubleshooting.Flakiness.md)

Reply:
Done